Python/self_printing_program/self_printing_program.py

```diff
@@ -0,0 +1,2 @@
+# A python program that prints its source code.
+print(open(__file__).read())
```

Comment:
A program can be called Quine only if it **takes no input at all**. In this case, the program reads its source code as a file, thus making it using itself as input. [src](https://towardsdatascience.com/how-to-write-your-first-quine-program-947f2b7e4a6f)

[Wikipedia](https://en.wikipedia.org/wiki/Quine_(computing)#:~:text=A%20quine%20is%20a%20computer,%22self%2Dcopying%20programs%22.)

Reply:
Absolutely correct AmirNaghibi. The issue remains open for Quine in python..